src/Client/ClientBase.cpp

```diff
@@ -1834,9 +1834,21 @@ bool ClientBase::executeMultiQuery(const String & all_queries_text)
 
 bool ClientBase::processQueryText(const String & text)
 {
-    if (exit_strings.end() != exit_strings.find(trim(text, [](char c) { return isWhitespaceASCII(c) || c == ';'; })))
+    auto trimmed_input = trim(text, [](char c) { return isWhitespaceASCII(c) || c == ';'; });
+
+    if (exit_strings.end() != exit_strings.find(trimmed_input))
         return false;
 
+    if (trimmed_input.starts_with("\\i"))
```

Comment:
Ok. But is it possible to group the code with others (`backslash_aliases`)?

Reply:
I wanted to do that at first, but switched to current way as `backslash_aliases` just rewrite input text while for `\i` need a different code for execution of queries so it seemed better to put this inside `processQueryText`.